2gp-demo/config/developer-scratch-def.json

```diff
@@ -1,16 +1,11 @@
 {
-  "orgName": "PLP Dev Perview Co",
+  "orgName": "PLP Dev Preview (2GP Developer)",
```

Comment:
currently, the links in the docs point to /config/project-scratch-def.json, and not /2gp-demo/config/developer-scratch-def.json/

we can modify this in the next doc patch, but in the meantime, needs two changes:
- all files that are nested under "2gp-demo" directory need to go back to home directory
- keep file name the same as before
- keep attributes like this with common names (i.e. remove "2GP Developer")

if the primary difference between the two types of configurations is mainly doc steps, then let's address that in the README by having two sections in the same readme, for now

Reply:
I'm incorporating these suggestions into a new branch today.